application/server/server_test.go

```diff
@@ -261,7 +261,7 @@ func Test_TextDocumentCodeLenses_shouldReturnCodeLenses(t *testing.T) {
 			path := uri.PathFromUri(didOpenParams.TextDocument.URI)
 			return workspace.Get().GetFolderContaining(path).DocumentDiagnosticsFromCache(path) != nil
 		},
-		5*time.Second,
+		50*time.Second,
```

Comment:
Why such a drastic change to the wait time?

Reply:
Because it might actually download the CLI

Reply:
And it wasn't previously? 👀 

Let's mark this as smoke test maybe then, since we have this category for long-running tests.